README.md

```diff
@@ -165,6 +165,100 @@ The format for an unstructured profile is below:
     }
 }
 ```
+# Profile Statistic Descriptions
+
+### Structured Profile
+
+#### global_stats:
+
+* `samples_used` - number of input data samples used to generate this profile
+* `column_count` - the number of columns contained in the input dataset
+* `row_count` - the number of rows contained in the input dataset
+* `row_has_null_ratio` - the proportion of rows that contain at least one null value to the total number of rows
+* `row_is_null_ratio` - the proportion of rows that are fully comprised of null values (null rows) to the total number of rows
+* `unique_row_ratio` - the proportion of distinct rows in the input dataset to the total number of rows
+* `duplicate_row_count` - the number of rows that occur more than once in the input dataset
+* `file_type` - the format of the file containing the input dataset (ex: .csv)
+* `encoding` - the encoding of the file containing the input dataset (ex: UTF-8)
+* `correlation_matrix` - matrix of shape `column_count` x `column_count` containing the correlation coefficients between each column in the dataset 
+* `chi2_matrix` - matrix of shape `column_count` x `column_count` containing the chi-square statistics between each column in the dataset
+* `profile_schema` - a description of the format of the input dataset labeling each column and its index in the dataset
+    * `string` - the label of the column in question and its index in the profile schema
+* `times` - the duration of time it took to generate the global statistics for this dataset in milliseconds
+
+#### data_stats:
+
+* `column_name` - the label/title of this column in the input dataset
+* `data_type` - the kind of data that is contained within this column
```

Comment:
kind of data is a little generic, should reference somehow that it is one of the primitive python data types